src/foam/u2/SplitScreen.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * NANOPAY CONFIDENTIAL
+ *
+ * [2021] nanopay Corporation
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of nanopay Corporation.
+ * The intellectual and technical concepts contained
+ * herein are proprietary to nanopay Corporation
+ * and may be covered by Canadian and Foreign Patents, patents
+ * in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from nanopay Corporation.
+ */
+
+ foam.CLASS({
+  package: 'foam.u2',
+  name: 'SplitScreen',
```

Comment:
Shouldn't this be called SplitScreenView?

Reply:
I just went with splitscreen because its not technically a foam 'view', but looking thru it we have some other non-'views' named WhateverView so maybe naming isnt strict as i thought
what do u think? i can change it to SplitScreenView as well

Reply:
Typically we would call this a Border so maybe we could use `SplitScreenGridBorder`

Reply:
cool that works for me, thanks